redhat-developer/odo
# add watch flag for create cmd

What is the purpose of this change? What does it change?

Adding `-w` `--watch` flag to `odo create` 

Was the change discussed in an issue?
https://github.com/redhat-developer/odo/issues/628

How to test changes?

`$odo create python --git https://github.com/openshift/django-ex.git`
 see if it exit quickly without showing logs


`odo create python -w --git https://github.com/openshift/django-ex.git`
see if this cmd streams the build log and wait for it to finish
